tools/CheckPythonFormatting.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Find all python files in src, support, tests, and tools, then run
+# them through yapf.
+if ! find ./src/ \
+     ./support/ \
+     ./tests/ \
+     ./tools/ \
+     -type f \
+     -name "*.py" \
+     -print0 \
+        | xargs --null yapf --parallel -d
```

Comment:
Could you change the `--null` to `-0` please? Then this command also works on macOS 👍